thirdparty/cmdconfig/commands/cmdsink/cmdsink.go

```diff
@@ -46,8 +48,18 @@ func (r *SinkRunner) runE(c *cobra.Command, args []string) error {
 	}
 	outputs := []kio.Writer{&kio.LocalPackageWriter{PackagePath: dir}}
 
+	f := kio.FilterFunc(func(nodes []*yaml.RNode) ([]*yaml.RNode, error) {
+		if err := kptfile.AreKRM(nodes); err != nil {
```

Comment:
at some point, define a filter that can be re-used in all places will be useful, I think.

Reply:
@phanimarupaka is making changes to `sink`, so this change might be affected by his changes.

Reply:
> at some point, define a filter that can be re-used in all places will be useful, I think.

will do